documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -58,246 +85,137 @@ base-char = 1-9 A-H J-N P-Z a-k m-z
-    * `diff` (REQUIRED): A Differentiation object containing all the changes compared to the DID Document it references in the `previousMessageId` field. The value of `diff` MUST be a JSON object following the [Anatomy of the Diff object](#anatomy-of-the-diff-object) definition.
-    * `proof` (REQUIRED): This field provides a cryptographic proof on the message that proves ownership over the DID Document. The value of the `proof` object MUST contain an object as defined by [Anatomy of the Proof object](#anatomy-of-the-proof-object).
-
-Example of a Diff DID message:
-```json
-{
-  "id": "did:iota:X7U84ez4YeaLwpfdnhdgFyPLa53twvAuMSYdRQas54e",
-  "diff": {
-    "doc": {
-      "service": [
-        {
-          "id": "did:iota:X7U84ez4YeaLwpfdnhdgFyPLa53twvAuMSYdRQas54e#linked-domain-1",
-          "type_": "LinkedDomains",
-          "service_endpoint": "https://example.com/"
-        }
+   "document":{
+      "id":"did:0:0",
+      "verificationMethod":[
+         {
+            "id":"did:0:0#key-1",
+            "controller":"did:0:0",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z6BNtMbKY78XDVuqfh4u15bZkByu94XNVr9RpqEGCNncn"
+         }
       ]
-    },
-    "meta": {
-      "updated": "2022-01-21T09:50:28Z"
-    }
-  },
-  "previousMessageId": "5dfff82c34c75b3436e7e03370e220e4693d39026fee315e6db7b7815305df4a",
-  "proof": {
-    "type": "JcsEd25519Signature2020",
-    "verificationMethod": "#sign-0",
-    "signatureValue": "3TG8LiXbWPcH3ecg4is5mswENVuQBakhv8R5TMXopHgPg578oLoczySvZMsEdjRHYgJGihK2VHEqyCHPjQyXW61m"
-  }
+   },
+   "metadata":{
+      "created":"2022-08-02T21:39:48Z",
+      "updated":"2022-08-02T21:39:48Z"
+   }
 }
 ```
 
-### Anatomy of the Proof object
-
-Following the proof format in the [Verifiable Credential standard](https://www.w3.org/TR/vc-data-model/#proofs-signatures), at least one proof mechanism, and the details necessary to evaluate that, MUST be expressed for a DID Document uploaded to the Tangle. 
+Note that `did:0:0` is a placeholder of the actual DID.
 
-The proof object is an embedded proof that contains all information to be verifiable. It contains one or more cryptographic proofs that can be used to detect tampering and verify the authorship of a DID creation or update. It mostly follows LD-Proofs standard.
+## Controllers
 
-**Type**
+A state controller can directly update the DID Document and the amount of coins held by the Alias Output, but it cannot destroy the output. A governor, on the other hand, can indirectly update the DID Document by updating the state controller. It can also destroy the output by performing a governance transition without producing an Alias Output with the same `Alias ID`.
```

Comment:
```suggestion
A state controller can directly update the DID Document and the amount of coins held by the Alias Output, but it cannot destroy the output. A governor, on the other hand, can indirectly update the DID Document by updating the state controller. They can also destroy the output by performing a governance transition without producing an Alias Output with the same `Alias ID`.
```

Reply:
The state controller can't destroy the Alias Output AFAIK. Only the governor can.

Reply:
You are absolutely right, this was meant as an attempt at more inclusive speech. Maybe we just go for `The governor can ...`.